clouds/cli/README.md

```diff
@@ -0,0 +1,21 @@
+# Cloud nuke CLI tools
+### AWS nuke
```

Comment:
I think it should be something like:

cloud-nuke <aws/azure> FLAGS

check https://click.palletsprojects.com/en/8.1.x/commands/#merging-multi-commands

Reply:
I updated the code so we can run both with poetry and `cloud-cli` executable.
The command are `cloud-cli <aws-nuke/azure-nuke> FLAGS` or `poetry run cloud/cli/cli.py <aws-nuke/azure-nuke> FLAGS`